packages/plugins/ArtBlocks/src/SNSAdaptor/Collectible.tsx

```diff
@@ -46,7 +53,12 @@ export function Collectible({ projectId, chainId: projectChainId }: CollectibleP
     const { value, loading, error } = useFetchProject(projectId, chainId)
     const project = value?.projects[0]
 
-    if (loading) return <LoadingBase />
+    if (!loading)
+        return (
+            <div className={classes.body}>
+                <LoadingBase />
+            </div>
+        )
```

Comment:
what about 
```suggestion
    if (loading) return <LoadingBase className={classes.body} />
```

Reply:
`LoadingBase` is not a semantic name. It's actually an animating loading icon. Applying such block style to it might not suitable.

Recommend using `<LoadingStatus />` instead. (You can omit loading text by setting `omitText=true`)